src/canister/individual_user_template/src/data_model/memory.rs

```diff
@@ -64,6 +65,10 @@ pub fn get_success_history_memory() -> Memory {
     MEMORY_MANAGER.with(|m| m.borrow_mut().get(SUCCESS_HISTORY_MEMORY))
 }
 
+pub fn get_success_history_v1_memory() -> Memory {
+    MEMORY_MANAGER.with(|m| m.borrow_mut().get(SUCCESS_HISTORY_MEMORY_V1))
+}
+
 pub fn init_memory_manager() {
```

Comment:
is there a reason for keeping older success history still? couldn't we reset the memory and store the new struct?